content/docs/2.4/concepts/scaling-deployments.md

```diff
@@ -121,6 +124,28 @@ This setting is passed to the HPA definition that KEDA will create for a given r
 
 ---
 
+```yaml
+  fallback:                                          # Optional. Section to specify fallback options
+    failureThreshold: 3                              # Mandatory if fallback section is included
+    replicas: 6                                      # Mandatory if fallback section is included
+```
+
+The `fallback` section is optional. It defines a number of replicas to fallback to if a scaler is in an error state.
+
+KEDA will keep track of the number of consecutive times each scaler has failed to get metrics from its source. Once that value passes the `failureThreshold`, instead of not propagating a metric to the HPA (the default error behaviour), the scaler will, instead, return a normalised metric using the formula:
+```
+target metric value * fallback replicas
+```
+Due to the HPA metric being of type `AverageValue` (see below), this will have the effect of the HPA scaling the deployment to the expected number of replicas.
+
+**N.B.** Fallback is only compatible with scalers that present their target as an `AverageValue` metric. For other scalers (CPU/Memory etc), KEDA will assume fallback is disabled.
```

Comment:
We should list all the not supported scalers instead, end-users don't know which ones use `AverageValue` metric since that's managed under the hood.

Reply:
I have added a suggestion below, let me know what you think.

Reply:
My worry with this change is twofold:
 - The etc covers custom use cases for the external scalers (which may or may not use `AverageValue`)
 - Explicitly adding unsupported (or indeed supported) scalers here, would be prone to becoming inaccurate over time; I imagine people PRing new scalers would forget to add to this list.

Thoughts?

Reply:
It would also be more explicit to end-users, though...

What do you think @zroubalik ?

Reply:
+1 for listing the scalers

Reply:
List becoming inaccurate over time is a valid concern, but I'd still take the risk